test.js

```diff
@@ -0,0 +1,7 @@
+function addNumbers() {
```

Comment:
Be sure to give your files names that make sense. Here, the file name "test.js" doesn't make sense - index, script, math, addition all would be fine.